libsoftwarecontainer/src/gateway/pulsegateway.cpp

```diff
@@ -23,20 +23,33 @@
 #include "pulsegateway.h"
 
 PulseGateway::PulseGateway() :
-    Gateway(ID)
+    Gateway(ID),
+    m_enableAudio(false)
 {
 }
 
 PulseGateway::~PulseGateway()
 {
 }
 
+bool PulseGateway::getConfValue(void)
+{
+    return m_enableAudio;
+}
```

Comment:
Too much effort for just one variable, but unfortunately you are right :)
i will do it